cmd/install/install.go

```diff
@@ -59,8 +59,11 @@ func Uninstall(cmd string) error {
 }
 
 func installers() (i []installer) {
-	if f := rcFile(".bashrc"); f != "" {
-		i = append(i, bash{f})
+	for _, rc := range [...]string{".bashrc", ".bash_profile"} {
```

Comment:
It looks like it is more correct to install in ~/.bash_profile directly if this is macOS.

~/.bashrc is not existent by default in a fresh macOS and some users seem to create ~/.bashrc without sourcing it from ~/.bash_profile